proprietary/tokens/src/components/ams/page-menu.tokens.json

```diff
@@ -3,12 +3,15 @@
     "page-menu": {
       "column-gap": { "value": "{ams.space.grid.md}" },
       "row-gap": { "value": "{ams.space.grid.xs}" },
+      "button": {
+        "cursor": { "value": "{ams.action.activate.cursor}" }
+      },
       "item": {
         "color": { "value": "{ams.link-appearance.color}" },
+        "column-gap": { "value": "{ams.space.xs}" },
         "font-family": { "value": "{ams.text.font-family}" },
         "font-size": { "value": "{ams.text.level.6.font-size}" },
```

Comment:
Not your change but level-6 looks weird compared to level-5 text right beneath it, especially in compact mode. Most of our testing has a spotlight or some large tekst under the menu so it did not stand out.